install/installer/traf_cloudera_mods

```diff
@@ -367,27 +370,37 @@ ssh -q -n $HDFS_NODE 'sudo su hdfs --command "' "$HADOOP_BIN_PATH"'/hdfs dfsadmi
 #       enabled and HDFS has been restarted
 echo "***INFO: Setting HDFS ACLs for snapshot scan support"
 ssh -q -n $HDFS_NODE 'sudo su hdfs --command "' "$HADOOP_BIN_PATH"'/hdfs dfs -mkdir -p /hbase/archive"'
-if [ $? != 0 ]; then
+errorFound=$?
+echo "ERROR Code: $errorFound"
```

Comment:
Do you want this echo here? Users will see a series of "ERROR Code: 0" messages in normal circumstances and wonder what all the fuss is about. Perhaps the value of $errorFound should be displayed inside the 'if' following?